### PR TITLE
fix checkboxes and video fps probe parsing

### PR DESCRIPTION
**Pull request by jmp909**
syncs args to UI
fixes incorrect use of os.sep in ffprobe output

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+core/__pycache__
```
Comment (henryruhs) [inline .gitignore:1]: remove your personal gitignores

File: core/utils.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def detect_fps(input_path):
     output = os.popen(f'ffprobe -v error -select_streams v -of default=noprint_wrappers=1:nokey=1 -show_entries stream=r_frame_rate "{input_path}"').read()
     if "/" in output:
         try:
-            return int(output.split(os.sep)[0]) // int(output.split(os.sep)[1]), output.removesuffix('\n')
+            return int(output.split("/")[0]) // int(output.split("/")[1]), output.removesuffix('\n')
```
Comment (henryruhs) [inline core/utils.py:26]: does not make sense to me... it's different on each os

Comment (jmp909) [reply core/utils.py:26]: @henryruhs  i thought that `os.sep` was just for file path separators? eg `c:\test` vs `/c/test` etc

this is not a file path it's a string output eg `15/15`

Comment (kex0) [reply core/utils.py:26]: As far as I understand, the output of `ffprobe -v error -select_streams v -of default=noprint_wrappers=1:nokey=1 -show_entries stream=r_frame_rate "{input_path}` is always some number followed by `/` and another number.
Example 
if fps is 30 then output is: `30/1`
if fps is 23.98 then output is `24000/1001`

I don't think the `/` is different in other OS, I might be wrong.

**Comment by henryruhs** [comment]
Merge to next branch

**Pull request merged**